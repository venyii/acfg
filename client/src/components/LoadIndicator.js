import React from 'react';

const LoadIndicator = () => (
  <div className="loader--container">
    <div className="loader" />
  </div>
);

export default LoadIndicator;
