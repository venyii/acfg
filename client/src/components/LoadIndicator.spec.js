import React from 'react';
import { shallow } from 'enzyme';
import LoadIndicator from './LoadIndicator';

describe('<LoadIndicator />', () => {
  it('renders a load indicator', () => {
    const wrapper = shallow(<LoadIndicator />);
    expect(wrapper).toMatchSnapshot();
  });
});
