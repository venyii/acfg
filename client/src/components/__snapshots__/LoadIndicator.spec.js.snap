// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadIndicator /> renders a load indicator 1`] = `
<div
  className="loader--container"
>
  <div
    className="loader"
  />
</div>
`;
